libs/neo4j/langchain_neo4j/vectorstores/neo4j_vector.py

```diff
@@ -928,6 +918,32 @@ def add_embeddings(
 
         return ids
 
+    def _build_import_query(self) -> str:
+        """
+        Build the Cypher import query string based on the Neo4j version.
+
+        Returns:
+            str: The constructed Cypher query string.
+        """
+        if self.neo4j_version_is_5_23_or_above:
```

Comment:
Where is this being set?

Reply:
https://github.com/langchain-ai/langchain-neo4j/blob/update-deprecated-cypher/libs/neo4j/langchain_neo4j/vectorstores/neo4j_vector.py#L672

Reply:
Thanks, wanted to make sure the aura versions work :)